ynput/ayon-aquarium
# Maintenance: Update repo structure

## Changelog Description
This PR aligns Repo's structure with the current expected structure by AYON.

## Demo
Here's how it looks after installing the addon to my AYON instance.
![image](https://github.com/ynput/ayon-aquarium/assets/20871534/138b6b37-a8b0-4a96-b6f0-af7efe780913)


## Additional Info
To Fully Test the addon, you'd need to 
1. Update your dependencies. using https://github.com/ynput/ayon-dependencies-tool
2. Install Services somehow on your server side. (Skip when Aquarium has its first released as AYON will be able to fetch the services)

## Additional Info 2
Client code doesn't work properly with AYON because of openpype calls. 
more info https://github.com/ynput/ayon-aquarium/issues/11

## Testing Notes
1. Run `python create_package.py`
3. Upload to Ayon Server, the addon should be recognized by the server
4. Feel free to test the Addon
